### DISCUSSION TITLE
ASP .NET MVC 6 Boilerplate Task Runner Failed

### DISCUSSION THREAD

**Discussion by phucan1108** [Dotnet-Boxed/Templates#49] [Q&A]
Hi,

I got a problem in Gulp Task Runner. When I create the MVC 6 Template, my project can run but all javascript/css files aren't loaded into the project. When I see Task Runner Explorer,  that shows "Failed to load. See output windows...". How could I fix that?. 

Thanks.

**Comment by RehanSaeed** [comment]
Take a look at the ReadMe. It explains how to fix this common issue near the top.
